schemas/client.yml

```diff
@@ -59,19 +59,35 @@
       description: >
         Client domain.
 
+    - name: name
+      level: core
+      type: keyword
+      short: Name of the client system.
+      description: >
+        Name of the client system.
+
+        It can contain what `hostname` returns on Unix systems, the fully
+        qualified domain name, or a name specified by the user. The sender
+        decides which value to use.
+        
+        In the case of fully and partially qualified domain names it is recommended to parse out 
+        the additional fields of top_level_domain, registered_domain, subdomain, and hostname
+        as they are available.
+      example: www.east.mydomain.co.uk 
+ 
     - name: registered_domain
       level: extended
       type: keyword
-      short: The highest registered client domain, stripped of the subdomain.
+      short: The highest registered domain, stripped of the subdomain.
       description: >
-        The highest registered client domain, stripped of the subdomain.
+        The highest registered domain, stripped of the subdomain.
 
         For example, the registered domain for "foo.google.com" is "google.com".
 
         This value can be determined precisely with a list like the public
         suffix list (http://publicsuffix.org). Trying to approximate this by
         simply taking the last two labels will not work well for TLDs such as "co.uk".
-      example: google.com
+      example: mydomain.co.uk
```

Comment:
Please keep the example aligned across the file when possible. In this case I agree with using .co.uk to make the point about `registered_domain`, but please adjust the description accordingly as well (line 85)

Reply:
I believe everything is now www.east.mydomain.com